docs/design/coreclr/jit/ryujit-overview.md

```diff
@@ -34,7 +34,7 @@ The following are the key methods on this interface:
   It returns a pointer to the code, its size, and additional GC, EH and (optionally) debug info.
   * `getVersionIdentifier` is the mechanism by which the JIT/EE interface is versioned.
   There is a single GUID (manually generated) which the JIT and EE must agree on.
-  * `getMaxIntrinsicSIMDVectorLength` communicates to the EE the largest SIMD vector length that the JIT can support.
+  * `getMaxVectorTBitWidth` communicates to the EE the the maximum width, in bits, that Vector<T> is allowed to be.
```

Comment:
FWIW, the reason for this API is historical. In the early days of `Vector<T>` support on .NET Framework, we have been shipping previews of the JIT with `Vector<T>` support as a separate download and one had to set the env variable to use it with standard .NET Framework bits included in the OS. It means that we needed a way to communicate to the VM what kind of `Vector<T>` support is in the custom JIT. 

This method does not serve its purpose anymore given how JIT and VM are developed in tandem these days. It would be fine to delete this method and have this tiny bit of logic in the VM.

Reply:
Removed